app/components/candidate_interface/application_visibility_component.rb

```diff
@@ -33,6 +33,7 @@ def offer?
   end
 
   def visible_to_providers?
-    application_form.candidate_pool_application.present?
+    application_form.candidate_pool_application.present? &&
```

Comment:
I would like to know what @CatalinVoineag thinks as he is closest to this work, but I think my preference would be to actually remove them from the pool if they are in it when they submit an application. https://github.com/DFE-Digital/apply-for-teacher-training/blob/0369feac420e659e2c95a7ddf1fd911d00e8e63e/app/services/candidate_interface/submit_application_choice.rb